### PR TITLE
Minor fixes

### PR DESCRIPTION
**Pull request by jmleroux**
Hello,
this repo has been very usefull to me thx.

Here are small fixes to stick with PHP standards:
- unused namespaces
- `__construct` does not return in PHP
- unused variables
- PHPDoc

### CODE REVIEW

**Comment by emcconville** [comment]
Thanks! I'll take a look shortly.

File: .travis.yml [hunk 1]
```diff
@@ -6,9 +6,11 @@ php:
   - hhvm
 
 before_script:
-  - composer self-update
-  - composer install --prefer-source --no-interaction --dev
- 
+  - if [ "$TRAVIS_PHP_VERSION" != "hhvm" ]; then
```
Comment (emcconville) [inline .travis.yml]: I'm dropping hhvm support, and the xdebug is needed for code-coverage.

File: src/Algorithms/Abstraction.php [hunk 2]
```diff
@@ -69,11 +67,11 @@ public function __construct( $points = null )
      *
      * @param mixed $points Must be array or Traversable
      *
-     * @return null
```
Comment (emcconville) [inline src/Algorithms/Abstraction.php:72]: Return is needed for docs, or code sniffer will emit a doc error

File: src/Algorithms/Abstraction.php [hunk 3]
```diff
@@ -204,7 +202,6 @@ protected function shortestDistanceToSegment(
         list( $ax, $ay ) = $a->getCoordinates();
         list( $bx, $by ) = $b->getCoordinates();
         list( $cx, $cy ) = $c->getCoordinates();
-        $return = 0.0;
```
Comment (emcconville) [inline src/Algorithms/Abstraction.php:207]: This is not an unused variable, but a prototype to hint at debuggers that `$return` is & always will be a double.

File: src/Algorithms/Abstraction.php (hunk 2 above)
Comment (jmleroux) [reply src/Algorithms/Abstraction.php:72]: weird, but if needed i'm ok.

Comment (emcconville) [reply src/Algorithms/Abstraction.php:72]: Totally weird. I'm completely indifferent if return should be set (in this case). BUT I like to do whatever the code document / lint program suggest. For this project, it's php's code_sniffer.

Please run `vendor/bin/phpcs src` on this branch, and resolve any issues emitted. Happy to accept this MR when both phpcs  + phpunit passes.

**Comment by jmleroux** [comment]
rebased and fixed

**Comment by emcconville** [comment]
Love it. Thanks!

**Pull request merged**